Aptoide/appcoins-wallet-android
# [feature/APPC-1080] remove hidden screen from payments flow

**What does this PR do?**

   This PR removes the hidden screen from the purchase flow. 

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] IabActivity.java


**How should this be manually tested?**
For each payment method check if there is a hidden screen appearing after choosing a payment method.

  Flow on how to test this or QA Tickets related to this use-case: [APPC-1080](https://aptoide.atlassian.net/browse/APPC-1080)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-1080](https://aptoide.atlassian.net/browse/APPC-1080)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)




**Code Review Checklist**
